server/netty-server/cats/src/main/scala/sttp/tapir/server/netty/internal/NettyCatsToResponseBody.scala

```diff
@@ -0,0 +1,99 @@
+package sttp.tapir.server.netty.internal
+
+import cats.effect.kernel.{Async, Sync}
+import cats.effect.std.Dispatcher
+import fs2.Chunk
+import fs2.interop.reactivestreams._
+import fs2.io.file.{Files, Flags, Path}
+import io.netty.buffer.Unpooled
+import io.netty.channel.ChannelHandlerContext
+import io.netty.handler.codec.http.{DefaultHttpContent, HttpContent}
+import org.reactivestreams.Publisher
+import sttp.capabilities.fs2.Fs2Streams
+import sttp.model.HasHeaders
+import sttp.tapir.server.interpreter.ToResponseBody
+import sttp.tapir.server.netty.NettyResponseContent.ByteBufNettyResponseContent
+import sttp.tapir.server.netty.{NettyResponse, NettyResponseContent}
+import sttp.tapir.{CodecFormat, RawBodyType, WebSocketBodyOutput}
+
+import java.io.InputStream
+import java.nio.ByteBuffer
+import java.nio.charset.Charset
+
+class NettyCatsToResponseBody[F[_]: Async](dispatcher: Dispatcher[F]) extends ToResponseBody[NettyResponse, Fs2Streams[F]] {
+  override val streams: Fs2Streams[F] = Fs2Streams[F]
+
+  override def fromRawValue[R](v: R, headers: HasHeaders, format: CodecFormat, bodyType: RawBodyType[R]): NettyResponse = {
+    bodyType match {
+      case RawBodyType.StringBody(charset) =>
```

Comment:
I think most branches are the same as in `NettyToResponseBody`? Maybe we can handle the different cases here, and for the raw body types that are the same as in `NettyToResponseBody`, delegate to that impl?